src/styles/blocks/menu.scss

```diff
@@ -0,0 +1,110 @@
+.menu {
+  background: #e5e5e5;
+  text-align: left;
+  z-index: 1000;
```

Comment:
Do you really need such a big z-index? If you do not have a large project, you can use smaller values (e. g. from 1 to 10)